src/ffi/spatialmath/mod.rs

```diff
@@ -1 +1,2 @@
-pub mod vector3;
\ No newline at end of file
+pub mod vector3;
+pub mod quaternion;
```

Comment:
I worry that I'm sounding like a broken record at this point, but I still think these files should all end in newlines, because not ending in a new line is not POSIX-compliant and some text editors aren't going to like that. I'd still be happy to help you configure your text editor to do this automatically, if you want. If you want to do it on your own, I think a decent starting point is https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Not a broken record; not ending in a newline is AFAIK explicitly against our internal style guide as well, and will break all sorts of things. Please make sure you keep calling this out wherever you see it.